journal-of-the-american-adacemy-of-audiology.csl

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Journal of the American Academy of Audiology</title>
+    <title-short>JAAA</title-short>
+    <id>http://www.zotero.org/styles/journal-of-the-american-adacemy-of-audiology</id>
```

Comment:
@damnation333, typo: "adacemy" should be "academy" (idem for file name and "self" link).